### PR TITLE
Start row Id column from 1

### PR DESCRIPTION
**Pull request by jungrishi**
Resolves: Start Row Id Column from one. 

Before:
start_from_0

After Fix:
start_from_1

### CODE REVIEW

File: good_first_issues/utils/__init__.py [hunk 1]
```diff
@@ -104,6 +104,17 @@ def identify_limit(limit: Optional[int], all: bool) -> Optional[int]:
         return 10
 
 
+def get_row_ids(issue_count, limiter):
+    """
+    Generate custom Row Index Column for the table
+    Returns a Iterable.
+    """
+    rowIDs = []
```
Comment (yankeexe) [inline good_first_issues/utils/__init__.py:112]: change to `row_ids` to match snake case variable naming convention of code base.

Comment (jungrishi) [reply good_first_issues/utils/__init__.py]: done

File: good_first_issues/utils/__init__.py [hunk 2]
```diff
@@ -104,6 +104,16 @@ def identify_limit(limit: Optional[int], all: bool) -> Optional[int]:
         return 10
 
 
+def get_row_ids(issue_count, limiter):
```
Comment (yankeexe) [inline good_first_issues/utils/__init__.py:107]: @jungrishi  I missed it on the earlier review, can you also add type annotation for the arguments and return type here. 

```python
def get_row_ids(issue_count: int, limiter: Optional[int]) -> Iterable[int]:
```

Also the the index from 1 needs to be implemented on the REST API output as well. 
https://github.com/yankeexe/good-first-issues/blob/2d7941c72d49e0b93d3ef217e192ecd957109878/good_first_issues/rest/commands.py#L74

Comment (jungrishi) [reply good_first_issues/utils/__init__.py:107]: sure

Comment (jungrishi) [reply good_first_issues/utils/__init__.py]: done

**Review by yankeexe — APPROVED** [review]

**Comment by yankeexe** [comment]
Thank you @jungrishi  🙂

**Pull request merged**